src/datadog.ts

```diff
@@ -6,12 +6,12 @@
  * Copyright 2021 Datadog, Inc.
  */
 
-import * as cdk from "@aws-cdk/core";
 import * as lambda from "@aws-cdk/aws-lambda";
+import * as cdk from "@aws-cdk/core";
 import { applyLayers, redirectHandlers, addForwarder, applyEnvVariables, defaultEnvVar } from "./index";
 import { Transport } from "./transport";
 
-export interface DatadogProps {
+export interface IDatadogProps {
```

Comment:
Do you know why this got renamed to `IDatadogProps`?

Reply:
Yeah this was because of an error from JSII: 
`src/datadog.ts:15:3 - error JSII3008: The "pythonLayerVersion" property of struct "datadog-cdk-constructs.DatadogProps" must be "readonly". Rename "datadog-cdk-constructs.DatadogProps" to "IDatadogProps" if it is meant to be a behavioral interface.`
I believe JSII has a naming convention for interfaces so I added an "I" to the front of the DatadogProps